packages/components/storybook/colors.stories.mdx

```diff
@@ -4,11 +4,13 @@ import { Meta, ColorPalette, ColorItem } from '@storybook/blocks'
 <Meta title="Design Tokens/Colors" />
 
 # Colors
+
```

Comment:
Autoformatting from VS code added the new lines + changed double quotes to single.